packages/app/src/app/pages/Search/Results/index.js

```diff
@@ -2,17 +2,30 @@ import React from 'react';
 
 import { Hits, Pagination } from 'react-instantsearch/dom';
 import Centered from 'common/lib/components/flex/Centered';
+import WideSandbox from 'common/lib/components/WideSandbox';
+import { sandboxUrl } from 'common/lib/utils/url-generator';
 
 import ResultInfo from '../ResultInfo';
-import SandboxCard from '../SandboxCard';
-
 import { Container } from './elements';
 
 function Results() {
   return (
     <Container>
       <ResultInfo />
-      <Hits hitComponent={SandboxCard} />
+      <Hits
+        hitComponent={({ hit }) => (
+          <WideSandbox
```

Comment:
It does seem a bit weird, after checking it seems like the description of the sandbox in search is ~150px and in explore ~130px height